solution/10-post-form/src/app.js

```diff
@@ -1,15 +1,12 @@
 const express = require('express');
 const path = require('path');
 const favicon = require('serve-favicon');
-// import 'body-parser' module
-const bodyParser = require('body-parser');
+
 
 const app = express();
 
 app.use(favicon(path.join(__dirname, '..', 'public', 'favicon.ico')));
 
-// parse incoming json
-app.use(bodyParser.json());
 // parse urlencoded bodies
 app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
Have you actually run the code after your changes? 

Running this file will result in a `ReferenceError` because you've removed line 5, where `bodyParser` is defined, yet still refer to it on line 14.